manifests/init.pp

```diff
@@ -92,51 +92,65 @@
   }
 
 
-  tomcat::install { 'tomcat':
-    package_name        => 'tomcat',
-    package_ensure      => 'present',
-    install_from_source => false,
-    user                => $tomcat_user,
-    group               => $tomcat_group,
-    manage_user         => false,
-    require             => User[$tomcat_user]
+  file {  '/opt/tomcat':
```

Comment:
All looks good. Potentially could add a variable for tomcat path to replace uses of literal string '/opt/tomcat' with $tomcat_path

Reply:
Addressed [35962b3](https://github.com/phac-nml/puppet-irida/pull/2/commits/35962b3c2c10a70115fee42a469290f8b04f9ef8)